src/cli/dedupe_v2.py

```
@@ -177,6 +183,63 @@ def scan(
+    logged — closing the TOCTOU window between dedupe scan and resolve.
+
+    On Windows (where SafeDir's POSIX primitives are unavailable) falls
+    back to a plain ``path.unlink()``.
+
+    Returns True if the file was removed, False on error or refusal.
+    """
+    if sys.platform == "win32":
+        try:
+            path.unlink()
+            renderer.render_resolve_action("removed", path)
+            return True
```

Ruff (0.15.13)

[warning] Consider moving this statement to an `else` block

(TRY300)